tests/bson/bson-binary-clone-001.phpt

```diff
@@ -0,0 +1,100 @@
+--TEST--
+MongoDB\BSON\Binary can be cloned
```

Comment:
Since you intentionally call `zend_objects_clone_members` in all of the BSON clone handlers, I wonder if there should be an extra test case that demonstrates an object with an application-assigned public property persists across cloning. This could be done by adding an extra line between the object creation and cloning within the `foreach` loop. For example:

```php
$binary = new MongoDB\BSON\Binary('randomBinaryData', $type);
$binary->foo = 'bar'; // Ensure zend_objects_clone_members is used
$clone = clone $binary;
```

Alternatively, an extra test after the `foreach` would do this as well. This approach would probably reduce the clutter from most of the `var_dump($clone)` calls and just let you have a final dump with the additional public property.

Reply:
Good point. I've added this. Note that these properties don't show up in `var_dump` output, but can be accessed normally on the clone. This also confirms that removing `zend_objects_clone_members` indeed stops copying those properties 😉

Reply:
Ah, no output in `var_dump()` means less test changes. And that makes sense since our debug handler only reads known properties from the struct. Works for me.